Makefile

```diff
@@ -234,7 +234,10 @@ mockgen: ## Install mockgen locally.
 KUSTOMIZE = $(shell pwd)/bin/kustomize
 .PHONY: kustomize
 kustomize: ## Download kustomize locally if necessary.
-	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v4@v4.5.7)
+	@if [ ! -f ${KUSTOMIZE} ]; then \
+		BINDIR=$(shell pwd)/bin ./hack/download-kustomize; \
```

Comment:
It compiles every time it runs, which is a lot more expensive than downloading the binary.

Reply:
But we are still using `go-get-tool` to install other packages. Why are we using it in some packages but not in all of them?

Reply:
IIRC kustomize used to be only available as source code and we had to compile. Now they provide binaries. 
I would be happy if we could download pre-built binaries everywhere instead of compiling on each CI run. Please feel free to open a PR.